mdbenchmark/mdengines/gromacs.py

```diff
@@ -17,9 +17,18 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with MDBenchmark.  If not, see <http://www.gnu.org/licenses/>.
+from __future__ import absolute_import
+
 import os
+import warnings
 from glob import glob
+from shutil import copyfile
+
+import click
+import datreant.core as dtr
+import mdsynthesis as mds
 import numpy as np
+from jinja2.exceptions import TemplateNotFound
```

Comment:
Do you specifically catch this exception somewhere? If not I would remove the import.